### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by icesar-spyro**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
```
Comment (fkuatspyro) [inline .github/ISSUE_TEMPLATE/bug_report.md:18]: Not relevant for scargo

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
```
Comment (fkuatspyro) [inline .github/ISSUE_TEMPLATE/bug_report.md:24]: We don't need screenshots

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+
+**Describe the solution you'd like**
+A clear and concise description of what you want to happen.
+
+**Describe alternatives you've considered**
+A clear and concise description of any alternative solutions or features you've considered.
+
+**Additional context**
+Add any other context or screenshots about the feature request here.
```
Comment (fkuatspyro) [inline .github/ISSUE_TEMPLATE/feature_request.md:20]: We don't need screenshots

**Review by fkuatspyro — CHANGES_REQUESTED** [review]

**Review by fkuatspyro — APPROVED** [review]

**Comment by icesar-spyro** [comment]
Updated the files by removing the irrelevant info

**Pull request merged**